packages/aws-cdk/lib/api/logs/logs-monitor.ts

```diff
@@ -200,9 +201,9 @@ export class CloudWatchLogEventMonitor {
       // if we have > 100 events let the user know some
       // messages have been supressed. We are essentially
       // showing them a sampling (10000 events printed out is not very useful)
-      if (response.nextToken) {
+      if (filteredEvents.length > 0 && response.nextToken) {
         events.push({
-          message: '(...messages supressed...)',
+          message: '(...messages suppressed...)',
```

Comment:
Can we change this to something like:

```suggestion
          message: '>>> `watch` shows only the first 100 log messages - the rest have been truncated...',
```

Feel free to word-smith the exact wording, of course.

Reply:
@skinny85 updated.